src/webgpu/shader/execution/robust_access.spec.ts

```diff
@@ -162,12 +162,25 @@ function* generateIndexableTypes({
     layout: scalarInfo.layout
       ? {
           alignment: scalarInfo.layout.alignment,
-          size: kArrayLength * arrayStride(scalarInfo.layout),
+          size:
+            storageClass === 'uniform'
+              ? // Uniform storage class must have array elements aligned to 16.
+                kArrayLength *
+                arrayStride({
+                  ...scalarInfo.layout,
+                  alignment: align(scalarInfo.layout.alignment, 16),
```

Comment:
Technically to match the `[[stride(16)]]` I think this would simply be `alignment: 16`?

Reply:
```suggestion
                  alignment: 16,
```